tests/ecosystem/upgrade/conftest.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='session')
```

Comment:
I think that it would be better to create both cephfs and rbd pods in the same fixture. You are doing that in the 'post_upgrade_pods' fixture

Reply:
ok, done